src/docs/examples.md

```diff
@@ -0,0 +1,44 @@
+# KGCL Commands.
+
+## `NodeRename`:
+	Command using CURIEs: `rename GO:0005635 from 'nuclear envelope' to 'foo bar'`
```

Comment:
did you mean to include the leading whitespace? This means quote as code in yaml

Reply:
This is markdown. YAML is separate.